excel/beginner/basics-of-excel/cells-continued.md

```diff
@@ -0,0 +1,78 @@
+---
+author: Stefan-Stojanovic
+
+levels:
+  - beginner
+
+aspects:
+  - new
+  - workout
+
+type: normal
+
+category: how to
+
+---
+
+# Cells (Find and Replace)
+
+---
+## Content
+
+Cells are essentially where all of your data goes. So finding and replacing mistakes should be easy.
+
+Excel, both Google docs and Office Excel provide a "find and replace" feature.

```

Comment:
Not sure I get this sentence. "Excel, Google Docs, and Office Excel"? Why are there 3? :D

Also you mean Google sheets, not Google Docs, right?

Reply:
Google sheets are just a part of google docs. The google sheets link is docs.google.com/spreadsheets
![image](https://user-images.githubusercontent.com/22899166/73239258-e53df280-419b-11ea-9f37-d4cfb6bae212.png) `Docs`, `Sheets`, `Slides` and `Forms` are Google Docs programs. Excel is a spreadsheet program on google docs, so google docs > spreadsheets. The Microsoft Excel is Office Excel, because it is a part of the Office 365 software and cannot be bought separately, so its Office Excel. This sentence states that Excel, on both versions of the software, has said function